hackathon/forms.py

```diff
@@ -182,3 +182,75 @@ def __init__(self, *args, **kwargs):
             self.fields['winning_project'] = forms.ModelChoiceField(
                 queryset=hack_projects)
             self.fields['winning_project'].required = False
+
+
+class EventForm(forms.ModelForm):
+    """
+    Form to create or update an Event 
+    """
+    title = forms.CharField(
+        label="Webinar Title",
+        widget=forms.TextInput(attrs={'class': 'form-control'})
+    )
+    start = forms.DateTimeField(
+        label="Start Time",
+        input_formats=['%d/%m/%Y %H:%M'],
+        widget=forms.DateTimeInput(
+            format='%d/%m/%Y %H:%M',
+            attrs={
+                'placeholder': 'DD/MM/YYYY HH:MM',
+                'autocomplete': 'off',
+                'class': 'form-control'
+            }
+        )
+    )
+    end = forms.DateTimeField(
+        label="End Time",
+        input_formats=['%d/%m/%Y %H:%M'],
+        widget=forms.DateTimeInput(
+            format='%d/%m/%Y %H:%M',
+            attrs={
+                'placeholder': 'DD/MM/YYYY HH:MM',
+                'autocomplete': 'off',
+                'class': 'form-control'
+            }
+        )
+    )
+    body = forms.CharField(
+        label="Description",
+        widget=forms.Textarea(attrs={'rows': 4, 'class': 'form-control'})
+    )
+    isReadOnly = forms.BooleanField(
+        label="Read Only",
+        required=False,
+        widget=forms.CheckboxInput(attrs={'class': 'form-check-input'})
+    )
+    webinar_link = forms.URLField(
+        label="Webinar Link",
+        required=False,
+        widget=forms.URLInput(attrs={'class': 'form-control'})
+    )
+    webinar_code = forms.CharField(
```

Comment:
I would not make this required or add instructions somewhere what to add in if there is no code.